libs/testdiff/golden.go

```diff
@@ -165,7 +169,7 @@ func PrepareReplacements(t testutil.TestingT, r *ReplacementsContext, w *databri
 	r.Set(w.Config.AzureResourceID, "$DATABRICKS_AZURE_RESOURCE_ID")
 	r.Set(w.Config.AzureClientSecret, "$ARM_CLIENT_SECRET")
 	// r.Set(w.Config.AzureClientID, "$ARM_CLIENT_ID")
-	r.Set(w.Config.AzureClientID, "$USERNAME")
+	r.Set(w.Config.AzureClientID, testerWithEmail)
```

Comment:
Why do we want to do this? It's not really accurate since I believe the azure client ids are uuids. Maybe we can instead replace this with a baseline uuid of `000000-00...`.